habitat/plan.ps1

```diff
@@ -16,9 +16,9 @@ $pkg_deps=@(
 )
 
 function Invoke-Begin {
-    $hab_version = (hab --version)
-    $hab_minor_version = $hab_version.split('.')[1]
-    if ( -not $? -Or $hab_minor_version -lt 85 ) {
+    $hab_version = (hab --version).split(" ")
+    [Version]$hab_version = $hab_version[1].split("/")[0]
```

Comment:
is putting these together too much for one line, i.e. `[Version]$hab_version = (hab --version).split(" ")[1].split("/")[0]`  ?

I kind of prefer that after poking at it.

otherwise I'd move the [1] up to the first line so the first time `$hab_version` is set it is really just a version number, i.e.

```
[String]$hab_version = (hab --version).split(" ")[1]
[Version]$hab_version = $hab_version.split("/")[0]
```

Reply:
yeah it can be one line I'll fix that